exchange/utils.go

```diff
@@ -263,6 +265,39 @@ func (rs *requestSplitter) cleanOpenRTBRequests(ctx context.Context,
 	return
 }
 
+// fpdUserEIDExists determines if req fpd config had User.EIDs
+func fpdUserEIDExists(req *openrtb_ext.RequestWrapper, fpd map[openrtb_ext.BidderName]*firstpartydata.ResolvedFirstPartyData, bidder string) bool {
+	fpdToApply, exists := fpd[openrtb_ext.BidderName(bidder)]
+	if !exists || fpdToApply == nil {
+		return false
+	}
+	if fpdToApply.User == nil {
+		return false
+	}
+	fpdUserEIDs := fpdToApply.User.EIDs
+	if len(fpdUserEIDs) == 0 {
+		return false
+	}
```

Comment:
Can we remove this statement? Technically it is possible that `EID` is specified as first party data as an empty array.
We can just lean on your length check and for loop below to make the determination.